source/client-side-encryption/tests/aggregate.template.yml

```diff
@@ -0,0 +1,61 @@
+runOn:
+  - minServerVersion: "4.1.10"
+database_name: &database_name "default"
+collection_name: &collection_name "default"
+
+data:
+  - &doc0_encrypted { _id: 1, ssn: {{ciphertext("457-55-5462", field="ssn")}} }
+json_schema: {{schema()}}
+key_vault_data: [{{key()}}]
+
+tests:
+  - description: "Aggregate with deterministic encryption"
+    skipReason: "SERVER-39395"
```

Comment:
Are there no aggregate tests that can pass yet?

Reply:
Even an empty pipeline would buy us some code coverage.

Reply:
Added a passing empty pipeline test. But until SERVER-39395 is in, I don't think there's much else I can test.